bin/flutter

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright 2015 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+FLUTTER_ROOT=$(dirname $(dirname "${BASH_SOURCE[0]}"))
+FLUTTER_TOOLS_DIR="$FLUTTER_ROOT/packages/flutter_tools"
+SNAPSHOT_PATH="$FLUTTER_ROOT/bin/cache/flutter_tools.snapshot"
+STAMP_PATH="$FLUTTER_ROOT/bin/cache/flutter_tools.stamp"
+SCRIPT_PATH="$FLUTTER_TOOLS_DIR/bin/sky_tools.dart"
+
+# TODO(abarth): We shouldn't require dart to be on the user's path.
+DART=dart
```

Comment:
We can download it to a known location, similar to how engine.git downloads the Dart SDK to `//third_parth/dart-sdk/dart-sdk`.